### ISSUE TITLE
Dependencies could not be resolved (1.0.0-beta.25 and 10.4.0)

### ISSUE THREAD

**Issue by maxhumber** [mapbox/mapbox-search-ios#38]
I am unable to resolve dependencies between the latest versions of MapboxMaps and MapboxSearch.

Swift Package File:

```swift
// swift-tools-version:5.5

import PackageDescription

let package = Package(
    name: "MyPackage",
    platforms: [.iOS(.v14)],
    products: [
        .library(name: "MyPackage", targets: ["MyPackage"]),
    ],
    dependencies: [
        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", from: "10.4.0"),
        .package(name: "MapboxSearch", url: "https://github.com/mapbox/mapbox-search-ios.git", from: "1.0.0-beta.25")
    ],
    targets: [
        .target(name: "MyPackage", dependencies: [
            "MapboxMaps",
            "MapboxSearch"
        ]),
        .testTarget(name: "MyPackageTests", dependencies: ["MyPackage"])
    ]
)
```

Error:

```
Dependencies could not be resolved because root depends on 'mapbox-maps-ios' 10.4.0..<11.0.0 and root depends on 'mapbox-search-ios' 1.0.0-beta.25..<2.0.0.
'mapbox-search-ios' is incompatible with 'mapbox-maps-ios' because 'mapbox-maps-ios' 10.4.0 depends on 'mapbox-common-ios' 21.2.0 and no versions of 'mapbox-maps-ios' match the requirement 10.4.1..<11.0.0.
'mapbox-search-ios' >= 1.0.0-beta.25 practically depends on 'mapbox-common-ios' 21.2.0-rc.1 because no versions of 'mapbox-search-ios' match the requirement 1.0.1..<2.0.0 and 'mapbox-search-ios' 1.0.0-beta.25 depends on 'mapbox-common-ios' 21.2.0-rc.1.
```

Please advise~

**Comment by sarochych** [comment]
Hi @maxhumber,
Yes, we had two separate releases of Maps and Search and both frameworks depended on a different version of the MapboxCommon.

Today we're going to make a new release of the Search to support Common of `21.2.0`. After that the issue should be resolved.

**Comment by sarochych** [comment]
So, the latest `MapboxCommon` support was added in #39 and should be available in `1.0.0-beta.26`.
Please, check this out.

**Comment by kalbaxa** [comment]
Closing now. Feel free to reopen if the issue still exists.
